src/ui-kit/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from '@redneckz/uni-jsx/lib/hooks';
+import type { LabelProps } from '../../model/HeadlineType';
+import type { IconVersion } from '../../model/IconVersion';
+import { formatDate } from '../../utils/formatDate';
+import { style } from '../../utils/style';
+import type { ControlProps } from '../ControlProps';
+import { Icon } from '../Icon/Icon';
+import type { IconName } from '../Icon/IconProps';
+import { Paragraph } from '../Paragraph/Paragraph';
+import { Select } from '../Select/Select';
+import { Text } from '../Text/Text';
+import { Calendar } from './Calendar';
+import { months } from './months';
+import { useDatePickerManager } from './useDatePickerManager';
+
+const ICON_PROPS: { name: IconName; iconVersion: IconVersion; width: string; height: string } = {
+  name: 'ArrowUpIcon',
+  iconVersion: 'black',
+  width: '24',
+  height: '24',
+};
+
+const SELECT_PROPS: { className: string; iconVersion: IconVersion; isBorder: boolean } = {
+  className: 'min-w-32',
+  iconVersion: 'black',
+  isBorder: false,
+};
+
+const formStyle =
+  'flex justify-between p-m border border-gray rounded-md cursor-pointer ' +
+  'hover:border-black active:border-black focus:border-black';
+
+interface DatePickerProps extends ControlProps<Date>, LabelProps {
+  className?: string;
+  minDate?: Date | undefined;
+  maxDate?: Date | undefined;
+}
+
+export const DatePicker = ({
+  className = '',
+  label = '',
+  value,
+  minDate,
+  maxDate,
+  onChange,
+}: DatePickerProps) => {
+  const [showCalendar, onShowCalendar] = useState(false);
```

Comment:
```suggestion
  const [showCalendar, setShowCalendar] = useState(false);
```

А лучше вообще `useBool` хук использовать из `uni-jsx`